src/components/CheckableButton/tests/CheckableButton.test.tsx

```diff
@@ -73,18 +72,5 @@ describe('<CheckableButton />', () => {
       element.find('div')!.trigger('onClick');
       expect(spy).toHaveBeenCalled();
     });
-
-    it('is called when the CheckableButton pressed with spacebar', () => {
-      const spy = jest.fn();
-      const element = mountWithApp(
-        <CheckableButton {...CheckableButtonProps} onToggleAll={spy} />,
-      );
-
-      element.find(Checkbox)!.find('input')!.trigger('onKeyUp', {
-        keyCode: Key.Space,
-      });
-
-      expect(spy).toHaveBeenCalled();
-    });
```

Comment:
Since we no longer manually handler the `keyUp` events, we no longer need to test this.

Reply:
Same here? Since KeyUp is back, does this still stand?